test/views/partials/card/payments/InCreditViewSpec.scala

```diff
@@ -44,6 +44,12 @@ class InCreditViewSpec extends ViewSpecBase {
       doc.getElementsByTag("p").first().text() mustBe "You are £123.45 in credit."
     }
 
+    "must have 'View your Corporation Tax statement' link" in {
+      doc.getElementById("view-ct-statement").text() mustBe "View your Corporation Tax statement"
+      doc.getElementById("view-ct-statement").attr("href", "http://localhost:8080/portal/corporation-tax/org/utr/account/balanceperiods?lang=eng")
```

Comment:
this is not a test condition, attr actually add/updates the attribute to the element.

this applies in other places as well

Reply:
Done.